### PR TITLE
Abhishek_Assignment_25_30minute

### PR DESCRIPTION
**Pull request by AbhishekSJune2020**
output:
Sum of all numbers in String is : 53

### CODE REVIEW

**Comment by PiyushPJun20** [comment]
Code looks good to me.
one suggestion, Instead of storing in Number variable, we can optimize code by using:
sum + = Integer.parseInt(String.valueOf(input.charAt(index)));

**Review by PiyushPJun20 — COMMENTED** [review]
Code looks good to me.
one suggestion, Instead of storing in Number variable, we can optimize code by using:
sum + = Integer.parseInt(String.valueOf(input.charAt(index)));

File: src/abhishekSJun20/SumOfNumbersInString.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	Output: Sum of all numbers in String is: 53
+ */
+
+public class SumOfNumbersInString {
+
+	void sumOfNumber(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```
Comment (KrishnaKTechnocredits) [inline src/abhishekSJun20/SumOfNumbersInString.java:13]: You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**